src/queryAnalysis/config/queryConfigs.go

```diff
@@ -0,0 +1,341 @@
+package config
+
+import "github.com/newrelic/nri-mssql/src/queryAnalysis/models"
+
+var Queries = []models.QueryDetailsDto{
```

Comment:
is regression testing is done , all the queries functionality wise working same as before?

Reply:
yes. Ran the queries on dev3 instance after making these changes and compared the result of old and new queries.
Results are same.